near-sdk/src/lib.rs

```diff
@@ -24,7 +24,7 @@
 //! near-sdk = "5.6.0"
 //! ```
 //!
-//! ### Example: Counter Smart Contract. For more information, see the [macro@near] documentation.
+//! ### Example: Counter Smart Contract. For more information, see the [near] documentation.
```

Comment:
can you duplicate this to README so it looks prominent there too ?
![image](https://github.com/user-attachments/assets/477e3dae-9404-471b-a542-cfd61e36987a)

Reply:
latest one

Reply:
Added. Please check I understood correctly :)